MomentumDiscordBot/Commands/Moderator/ModeratorCustomModule.cs

```diff
@@ -0,0 +1,138 @@
+            {
+                await Config.SaveToFileAsync();
+                await context.Client.GetSlashCommands().RefreshCommands();
+                await ReplyNewEmbedAsync(context, $"Command '{name}' added", MomentumColor.Blue);
+            }
+            else
+                await ReplyNewEmbedAsync(context, "Failed to add command", MomentumColor.Red);
+        }
+
+        [SlashCommand("remove", "deletes a custom commands")]
+        public async Task RemoveCustomCommandAsync(InteractionContext context
+        , [ChoiceProvider(typeof(CustomCommandChoiceProvider))] [Option("name", "name of the custom command")] string name)
+        {
+            CustomCommand command;
+            if(Config.CustomCommands.TryRemove(name, out command))
+            {
+                await Config.SaveToFileAsync();
+                await context.Client.GetSlashCommands().RefreshCommands();
+                await ReplyNewEmbedAsync(context, $"Command '{name}' removed", MomentumColor.Blue);
+            }
+            else
+            {
+                await ReplyNewEmbedAsync(context, "Failed to remove command.", MomentumColor.Blue);
+            }
+        }
+
+        [SlashCommand("rename", "deletes a custom commands")]
+        public async Task RenameCustomCommandAsync(InteractionContext context
+        , [ChoiceProvider(typeof(CustomCommandChoiceProvider))] [Option("oldName", "name of the custom command")] string oldName
+        , [Option("newName", "the new name")] string newName)
+        {
+            if(Config.CustomCommands.ContainsKey(newName))
+                    await ReplyNewEmbedAsync(context, "Command '" + newName + "' already exists!", MomentumColor.Red);
+
+            else if(Config.CustomCommands.ContainsKey(oldName))
+            {
+                CustomCommand command;
+                if(!Config.CustomCommands.TryGetValue(oldName,out command))
+                    await ReplyNewEmbedAsync(context, "Failed to get old command value.", MomentumColor.Red);
+                else if(!Config.CustomCommands.TryAdd(newName, command))
+                    await ReplyNewEmbedAsync(context, "Failed to add new command.", MomentumColor.Red);
+                else if(!Config.CustomCommands.TryRemove(oldName, out command))
+                    await ReplyNewEmbedAsync(context, "Failed to remove old command.", MomentumColor.Red);
+                else
+                {
+                    await Config.SaveToFileAsync();
+                    await context.Client.GetSlashCommands().RefreshCommands();
+                    await ReplyNewEmbedAsync(context, $"Command '{oldName}' was renamed to '{newName}'", MomentumColor.Blue);
+                }
+            }
+        }
+
+        [SlashCommand("list", "lists all custom commands")]
+        public async Task ListCustomCommandAsync(InteractionContext context)
+        {
+            var embedBuilder = new DiscordEmbedBuilder
+            {
+                Title = "Commands:",
```

Comment:
Colon looks a bit weird here, can just remove it. And maybe "Info Commands" is better.